Makefile

```diff
@@ -10,27 +13,28 @@ all: examples prepare-for-rootfs
 clean: examples-clean prepare-for-rootfs-clean
 
 examples:
-	for example in $(EXAMPLE_LIST); do \
-		$(MAKE) -C $$example CROSS_COMPILE="$(HOST_CROSS_COMPILE)"; \
+	@for example in $(EXAMPLE_LIST); do \
+		$(MAKE) -C $$example CROSS_COMPILE="$(HOST_CROSS_COMPILE)" || exit -1; \
 	done
 
 examples-clean:
-	for example in $(EXAMPLE_LIST); do \
+	@for example in $(EXAMPLE_LIST); do \
 		$(MAKE) -C $$example clean; \
 	done
 
 prepare-for-rootfs: examples
-	echo "Copying example CA and TA binaries to $(OUTPUT_DIR)..."
+	@echo "Copying example CA and TA binaries to $(OUTPUT_DIR)..."
 	@mkdir -p $(OUTPUT_DIR)
 	@mkdir -p $(OUTPUT_DIR)/ta
 	@mkdir -p $(OUTPUT_DIR)/ca
 	@for example in $(EXAMPLE_LIST); do \
-		if [ -e $$example/host/optee_$$example ]; then \
-			cp -p $$example/host/optee_$$example $(OUTPUT_DIR)/ca/; \
+		if [ -e $$example/host/optee_example_$$example ]; then \
+			cp -p $$example/host/optee_example_$$example $(OUTPUT_DIR)/ca/; \
 		fi; \
 		cp -pr $$example/ta/*.ta $(OUTPUT_DIR)/ta/; \
 	done
 
 prepare-for-rootfs-clean:
-	rm -rf $(OUTPUT_DIR)/ta/*
-	rm -rf $(OUTPUT_DIR)/ca/*
+	@rm -rf $(OUTPUT_DIR)/ta/*
```

Comment:
What's the purpose of the `*` at the end?

Reply:
hmmm! none. i'll clean. thanks.